states/state.py

```diff
@@ -18,6 +20,8 @@ def list_options(cls: Type[T]) -> List[str]:
 
 
 class State:
+    """Function for main state and error catcher"""
+
     @staticmethod
     def check_input(option_type: Type[T]) -> T:
```

Comment:
Don't forget to document class methods

Reply:
Added description `Returns the the options as a string for application to print to user` to classmethod and `Receives user input runs application accordingly even in the case an option not available is chosen` tp staticmethod

Reply:
`Receives user input runs application accordingly even in the case an option not available is chosen` is a bit long, so consider chopping off the rest of the sentence starting at `even`. It's good documentation but I think the finer details can be discovered if need be.

Reply:
Also, minor: `Returns the the options`

Reply:
Changed static method description to `Receives user input runs application accordingly` and stopped stuttering in the classmethod description